modules/samples/artifacts/ReceiveHTTPInJsonFormatWithCustomMapping/ReceiveHTTPInJsonFormatWithCustomMapping

```diff
@@ -0,0 +1,72 @@
+@App:name("ReceiveHTTPInJsonFormatWithCustomMapping")
+
+@App:description('Receive events via HTTP transport in JSON format with custom mapping and view the output on the console')
+/*
+Purpose:
+    This application demonstrates how to configure WSO2 Stream Processor to receive sweet production events via HTTP transport in JSON format using custom mapping and log the  events in LowProducitonAlertStream to the  output  console.
```

Comment:
Can we rephrase the to say receive events to the SweetProductionStream instead of sweet production events?
Also check for double spaces